flask_toolkit/infra/module.py

```diff
@@ -0,0 +1,65 @@
+import importlib
+
+
+class AutoDiscover:
+    """
+    This class search modules from path recursively. What it does
+    is actually search all the modules and run an `import_module` for each.
+
+    :param path: `pathlib.Path` object
+    :param pattern: Must be a string for search likes string.endswith(pattern)
+    :return: modules list
+
+    How to use:
+
+        autodiscover = AutoDiscover(path=...)
+        autodiscover()
+    """
+
+    def __init__(self, path, pattern=None):
+        self.path = path
+        self.pattern = pattern
+        self.root = self.__get_dotted_full_path(path=path)
+
+    def __call__(self):
+        return self.__autodiscover(path=self.path, pattern=self.pattern)
+
+    def __autodiscover(self, path, pattern):
+        modules = []
+
+        for obj in path.iterdir():
+            if obj.name.startswith('_'):
+                continue
+
+            if (
+                    obj.is_file()
```

Comment:
:nail_care: Tem uns espaços a mais aqui... acho que da pra alinhar o `obj` com `and`.

Reply:
hahaha eu deixei para alinhar a condição... mas posso voltar também :tada: